modules/admin_manual/pages/maintenance/upgrade.adoc

```diff
@@ -49,4 +49,4 @@ in conjunction with our official ownCloud repositories. *Note:* This approach sh
 nor in clustered setups.
 We discourage upgrades with Linux Package Manager because you might encounter unwanted side effects.
 
-NOTE: Enterprise customers will use their Enterprise software repositories to maintain their ownCloud servers, rather than the Open Build Service. Please see xref:enterprise/installation/install.adoc[Installing & Upgrading ownCloud Enterprise Edition] for more information.
+NOTE: Enterprise customers will refer to Installing & Upgrading ownCloud Enterprise Edition for more information. Please see xref:enterprise/installation/install.adoc[Installing & Upgrading ownCloud Enterprise Edition] for more information.
```

Comment:
Redundant info here? It should be:

NOTE: Enterprise customers will refer to xref:enterprise/installation/install.adoc[Installing & Upgrading ownCloud Enterprise Edition] for more information.

Reply:
Agreed.